src/project.js

```diff
@@ -23,4 +24,4 @@ var useWith = require('./useWith');
  *      var kids = [abby, fred];
  *      R.project(['name', 'grade'], kids); //=> [{name: 'Abby', grade: 2}, {name: 'Fred', grade: 7}]
  */
-module.exports = useWith(_map, [pickAll, identity]); // passing `identity` gives correct arity
+module.exports = useWith(_map, [pickWithDefault(__, undefined), identity]); // passing `identity` gives correct arity
```

Comment:
This suggests that the default value should be the first argument rather than the second.